src/file-utils/tests/test_file_reader.py

```diff
@@ -1,299 +1,325 @@
-        )
-        mocker.patch(
-            "edfi_lms_file_utils.file_repository.get_submissions_file",
-            lambda _a, _b, _c: None,
-        )
-        mocker.patch(
-            "edfi_lms_file_utils.file_repository.get_attendance_events_file",
-            lambda _a, _b: None,
-        )
-
-    def describe_when_getting_users():
-        def it_should_return_an_empty_DataFrame(mocker, fixture):
-            df = get_all_users(BASE_DIRECTORY)
-
-            assert df.empty
-
-    def describe_when_getting_section():
-        def it_should_return_an_empty_DataFrame(mocker, fixture):
-            df = get_all_sections(BASE_DIRECTORY)
-
-            assert df.empty
-
-    def describe_when_getting_system_activities():
-        def it_should_return_an_empty_DataFrame(mocker, fixture):
-            df = get_all_system_activities(BASE_DIRECTORY)
-
-            assert df.empty
-
-    def describe_when_getting_section_associations():
-        def it_should_return_an_empty_DataFrame(mocker, fixture):
-
-            df = get_all_section_associations(BASE_DIRECTORY, INPUT_DF)
-
-            assert df.empty
-
-    def describe_when_getting_section_activities():
-        def it_should_return_an_empty_DataFrame(mocker, fixture):
-
-            df = get_all_section_activities(BASE_DIRECTORY, INPUT_DF)
-
-            assert df.empty
-
-    def describe_when_getting_assignments():
-        def it_should_return_an_empty_DataFrame(mocker, fixture):
-
-            df = get_all_assignments(BASE_DIRECTORY, INPUT_DF)
-
-            assert df.empty
-
-    def describe_when_getting_submissions():
-        def it_should_return_an_empty_DataFrame(mocker, fixture):
-
-            df = get_all_submissions(BASE_DIRECTORY, INPUT_DF)
-
-            assert df.empty
-
-    def describe_when_getting_grades():
-        def it_should_return_an_empty_DataFrame(mocker, fixture):
-
-            df = get_all_grades(BASE_DIRECTORY, INPUT_DF)
-
-            assert df.empty
-
-    def describe_when_getting_attendance_events():
-        def it_should_return_an_empty_DataFrame(mocker, fixture):
-
-            df = get_all_attendance_events(BASE_DIRECTORY, INPUT_DF)
-
-            assert df.empty
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+import pandas as pd
+from unittest.mock import Mock
+import pytest
+
+from edfi_lms_file_utils.file_reader import (
+    _read_csv, get_all_users,
+    get_all_sections,
+    get_all_section_associations,
+    get_all_section_activities,
+    get_all_system_activities,
+    get_all_assignments,
+    get_all_submissions,
+    get_all_grades,
+    get_all_attendance_events,
+    read_file_from_path,
+)
+from edfi_lms_file_utils.constants import Resources, DataTypes
+from .constants import BASE_DIRECTORY
```

Comment:
I didn't like the idea of having one test importing constants from another test, so I moved the base directory to a constants file and imported it from both test files